src/Microsoft.Data.SqlClient/netfx/src/Resources/Strings.Designer.cs

```diff
@@ -39,7 +39,7 @@ internal class Strings {
         internal static global::System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
-                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.Data.SqlClient.Resources.Strings", typeof(Strings).Assembly);
+                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("SqlClient.Resources.Strings", typeof(Strings).Assembly);
```

Comment:
Failures should came from these change!
```suggestion
                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("Microsoft.Data.SqlClient.Resources.Strings", typeof(Strings).Assembly);
```

Reply:
Fixed. The resource naming needs to be looked at sometime in a far future when we have time for such small issues.

Reply:
Seems it hasn't been fixed! 😉

Reply:
Nope. And I don't understand why it's broken or how i broke it in this PR. I may have to close and start again on a clean branch.